src/Symfony/Component/Security/Http/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.3
+---
+
+ * Improve `LogoutUrlGenerator::getListener()` exception message when no listener found
```

Comment:
I don't think we need a CHANGELOG entry for this change as it is not something actionable.

Reply:
I am also in favor of merging this in `6.2` 👍